frontend/BOILERPLATE.md

```diff
@@ -0,0 +1,72 @@
+# Next.js redux starter
```

Comment:
Please do not add copy-pasted files that do not belong here.

Reply:
Will remove. It's part of SSR boilerplate. (implementing SSR on your own can be very buggy)